Integer/integer.cpp

```diff
@@ -42,6 +48,43 @@ Integer &Integer::operator% (const Integer &value)
     return *this;
 }
 
-bool Integer::operator< (const Integer &value){
+bool Integer::operator< (const Integer &value)
+{
 	return (getValue() < value.getValue());
 }
+
+bool Integer::operator> (const Integer &value)
+{
+	return (getValue() > value.getValue());
+}
+
+bool Integer::operator<= (const Integer &value)
+{
+	return (getValue() <= value.getValue());
+}
+
+bool Integer::operator>= (const Integer &value)
+{
+	return (getValue() >= value.getValue());
+}
+
+bool Integer::operator== (const Integer &value)
+{
+	return (getValue() == value.getValue());
+}
+
+Integer &Integer::operator+ (const Integer &value)
```

Comment:
Sergey
Just try to do
Integer v1, v2;
v1 = 1;
v2 = v1 + 9;
//What is value of v1? Is it 1? No - it's 10. v1 == v2 here.